docs/included.md

```diff
@@ -431,6 +431,14 @@
     [Tutorial](project:./tutorials/Binary_Contingency_Scores.md)
   - 
     [Wikipedia](https://en.wikipedia.org/wiki/Precision_and_recall); [Success ratio (WWRP/WGNE Joint Working Group on Forecast Verification Research)](https://www.cawcr.gov.au/projects/verification/#SR)
+* -  
+    - Positive Predictive Value (Success Ratio)
```

Comment:
```suggestion
    - Positive Predictive Value (Success Ratio, Precision)
```

Reply:
Also, the "Success Ratio" and "Precision" entries in included.md will both need updating to include "Positive Predictive Value" as an alternate term in the brackets. (Hope my explanation makes sense - if not, please feel free to either ask @tennlee or me for further clarification)